controllers/osccluster_bastion_controller.go

```diff
@@ -351,15 +351,13 @@ func reconcileDeleteBastion(ctx context.Context, clusterScope *scope.ClusterScop
 		return reconcile.Result{}, err
 	}
 
-	var securityGroupIds []string
 	bastionSecurityGroups := clusterScope.GetBastionSecurityGroups()
 	for _, bastionSecurityGroup := range *bastionSecurityGroups {
 		securityGroupName := bastionSecurityGroup.Name + "-" + clusterScope.GetUID()
-		securityGroupId, err := getSecurityGroupResourceId(securityGroupName, clusterScope)
+		_, err := getSecurityGroupResourceId(securityGroupName, clusterScope)
```

Comment:
Can you seperate commits please ?

Reply:
I don't understand why.
the `securityGroupIds` value is never used and the same for `securityGroupId`

Reply:
I meant since this commit is not part of the golangci lint fixes, I prefere it to be in a seperate commit

Reply:
I fixed this one in #370 so we can remove it here